charts/fluent-operator/values.yaml

```diff
@@ -387,7 +387,10 @@ fluentbit:
   parsers:
     javaMultiline:
     # use in filter for parser generic springboot multiline log format
-      enable: false
+      enable:
```

Comment:
the `enable: false` shouldn't be changed

Reply:
Reverted enable: false value.

Reply:
Could you please review this again?